jnigen/test/jackson_core_test/third_party/lib/_init.dart

```diff
@@ -0,0 +1,22 @@
+// Generated from jackson-core which is licensed under the Apache License 2.0.
```

Comment:
This file actually doesn't contain any code from the original header file, so it doesn't necessarily need the same copyright. But since it is generated, maybe it should just have the identical preamble to the other files.

Or is it not generated?

Reply:
Generated.